Installer/SetupSILConverters/SetupSILConverters.wixproj

```diff
@@ -17,10 +17,10 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
     <OutputPath>bin\$(Platform)\$(Configuration)\</OutputPath>
     <IntermediateOutputPath>obj\$(Platform)\$(Configuration)\</IntermediateOutputPath>
-    <DefineConstants>ApplicationName=SILConverters;SafeApplicationName=SILConverters;TruncatedVersion=5.1;VersionNumber=5.1.1.0;UpgradeCode=8A50226C-84D0-4CAF-AF8F-B0284643C415;Year=2022;Manufacturer=SIL International;EcDistFilesPath=$(EcDistFilesPath)</DefineConstants>
+    <DefineConstants>ApplicationName=SILConverters;SafeApplicationName=SILConverters;TruncatedVersion=5.1;VersionNumber=5.1.2.0;UpgradeCode=8A50226C-84D0-4CAF-AF8F-B0284643C415;Year=2022;Manufacturer=SIL International;EcDistFilesPath=$(EcDistFilesPath)</DefineConstants>
```

Comment:
`Year=2022` There has to be a way to centralize some of these constants.

Reply:
if you have a suggestion, I'm open. I found out about having a "Directory.Build.props" file in the solution root and I do have that (read: I have to change it in both places). But for some reason, they aren't visible in wix projects where these are needed.

Reply:
Hmm. In [genericinstaller](https://github.com/sillsdev/genericinstaller/blob/master/BaseInstallerBuild/buildBaseInstaller.bat), we pass this on the command line, but your setup is probably different.

I've tried moving <DefineConstants> out of the <PropertyGroup Condition=Release|x86>, but if you edit that in VisualStudio, you'll probably be right back where you started.

Did you try putting <DefineConstants> in Directory.Build.props, or only <Year>2023</Year>? But if <DefineConstants>Year=$(Year)</DC> didn't work, I'm out of ideas.

Reply:
yeah... it just doesn't want to go from Directory.Build.props to a wix project w/ wix 3.11.2... there is a v4 of wix, but I couldn't upgrade, bkz "Package 'wix 4.0.0' has a package type 'DotnetTool' that is not supported by project 'Installer\SEC Setup'" (which I think is looking for .net core functionality which this code base doesn't support). Anyway, these defines are defined in the VS UI and stored in the proj file, but as long as I remember to change them in both places (x86 and x64, which have different upgrade codes anyway), then it works okay. 

maybe a project for another release (when I have more time to research it)